htdocs/modules/system/include/update.php

```diff
@@ -105,17 +105,13 @@ function xoops_module_update_system(&$module, $oldversion = NULL, $dbVersion = N
 
 	/* Begin upgrade to version 1.5 */
 	if (!$abortUpdate) {
-		$newDbVersion = 47;
+		$newDbVersion = 48;
 	}
```

Comment:
I changed this because I was running the beta version on my site. I didn't think to add another section. I can if you think it best

Reply:
I have created separate blocks for beta 1 and beta 2 - this should be resolved.